src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/EdgeCasesTests.cs

```diff
@@ -279,9 +279,10 @@ public static void EnvelopedCmsEncryptWithNullRecipients()
         }
 
         [Fact]
+        // On the desktop, this throws up a UI for the user to select a recipient. We don't support that.
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
If it works on netcore it'll be the corefx implementation, so only netfx should be skipped.